### ISSUE TITLE
bug: cannot start front-end on fresh local instance

### ISSUE THREAD

**Issue by introkun** [TreyWW/MyFinances#239]
### What version of MyFinances are you currently using?

HEAD

### What device type are you currently facing the issue on?

Desktop

### Describe the bug

```
❯ npm run tailwind-build


> myfinances@0.2.1 tailwind-build
> tailwindcss -i .\\frontend\\static\\src\\input.css -o .\\frontend\\static\\src\\output.css -w

Specified input file .\frontend\static\src\input.css does not exist.
```

**Comment by introkun** [comment]
Path is not correct for Linux/MacOs

**Comment by TreyWW** [comment]
Oh right. Yeah I don't know why I even changed that to be fair, I was trying to fix a few issues and forgot to remove it lol. I'll fix that now. Sorry :)

**Comment by TreyWW** [comment]
Should be fixed in 60f360e. Let me know if it didn't fix it.

Thanks!

**Comment by introkun** [comment]
oh, I did some updates too :) https://github.com/TreyWW/MyFinances/pull/240
